src/components/Input/Input.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+
+import { useState } from 'react';
+import { InputType } from '~/types';
+import WebsiteInputField from './InputField/WebsiteInputField';
+import SongInputField from './InputField/SongInputField';
+import TextInputField from './InputField/TextInputField';
+import { fetchArticleData } from '~/query/fetch-article-data';
+import Container from '../utility-components/Container';
+
+const Input = () => {
+	const [inputTypeSelected, setInputTypeSelected] = useState<InputType>(
+		InputType.WEBSITE
+	);
+	const [summaryLength, setSummaryLength] = useState('200');
+
+	const handleFormSubmit = async (
+		event: React.SyntheticEvent,
+		inputUrl: string
+	) => {
+		event.preventDefault();
+		try {
+			const res = await fetchArticleData(inputUrl);
+
+			const existingData = localStorage.getItem('summaries');
+			if (existingData) {
+				const existingDataArr: { [key: string]: string }[] =
+					JSON.parse(existingData);
+				existingDataArr.push(res);
+				console.log(JSON.stringify(existingDataArr));
+				localStorage.setItem('summaries', JSON.stringify(existingDataArr));
+			} else {
+				console.log(JSON.stringify([res]));
+				localStorage.setItem('summaries', JSON.stringify([res]));
+			}
+		} catch (error) {
+			console.error(error);
+		}
+	};
+
+	return (
+		<Container>
+			<div className='mx-auto max-w-[31.875rem] py-8 text-center'>
+				<div className='mx-auto inline-flex h-12 items-center justify-center rounded-full bg-dark p-1 text-white'>
+					{(Object.keys(InputType) as Array<keyof typeof InputType>).map(
+						(key) => (
+							<button
+								key={key}
+								className={`h-full min-w-[6.625rem] rounded-full transition${
+									InputType[key] === inputTypeSelected && ' bg-white text-dark'
+								}`}
+								onClick={() => setInputTypeSelected(InputType[key])}>
+								{InputType[key]}
+							</button>
+						)
+					)}
+				</div>
+				<div className='my-12 flex flex-col items-center justify-center'>
+					<label htmlFor='summary-length' className='mb-6 font-bold text-dark'>
+						Length of the summary in words:
+					</label>
+					<input
+						type='range'
+						className='flex min-w-[30rem] cursor-pointer items-center bg-medium p-0 focus:shadow-none'
```

Comment:
We may want to adjust the width for mobile here so that mobile views don't overflow.